src/rank_llm/analysis/response_analysis_verbose.py

```diff
@@ -120,17 +162,35 @@ def count_errors(
 
 
 def main(args):
-    response_analyzer = ResponseAnalyzer(args.files)
-    responses, num_passages = response_analyzer.read_saved_responses()
-    print("Normalized scores:")
+    if args.files:
+        input_data = args.files
+    elif args.input_json_analyzer:
+        input_data = load_json_input_analyzer(args.input_json_analyzer)
+    else:
+        raise ValueError("Either --files or --input_json must be provided.")
+
+    response_analyzer = ResponseAnalyzer(input_data)
+    responses, num_passages = response_analyzer.read_responses()
 
-    print(response_analyzer.count_errors(responses, num_passages, args.verbose))
     # Print normalized scores
+    print("Normalized scores:")
+    print(response_analyzer.count_errors(responses, num_passages, args.verbose))
 
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
-    parser.add_argument("--files", type=str, nargs="+", required=True)
+    parser.add_argument("--files", type=str, nargs="+", required=False)
+    parser.add_argument(
```

Comment:
no need to support serialized json results here, there won't be any json file containing serialized results. the files in the argument above is the serialized ranking_exec_summaries. What I want to add is analyzing list[results] in the demo, without having to write them to a file first